test/test_controllers_clouds_gui.py

```diff
@@ -91,7 +91,9 @@ def test_finish_w_controller(self):
         self.mock_gcc.return_value = 'testcontroller'
         self.controller.finish('testcloud')
         self.mock_juju.assert_has_calls([
-            call.switch_controller('testcontroller')])
+            ANY,
+            ANY,
```

Comment:
What are the first two calls here? if possible we should test them, or else just avoid asserting anything about them, and use assert_called_once_with or something